projects/prison-custody-status-to-delius/src/main/resources/application.yml

```diff
@@ -16,9 +16,33 @@ spring:
           global_temporary:
             create_tables: false
             drop_tables: false
+  security.oauth2.client:
+    registration:
+      prison-custody-status-to-delius:
+        provider: hmpps-auth
+        authorization-grant-type: client_credentials
+        client-id: ${oauth2.client-id}
+        client-secret: ${oauth2.client-secret}
```

Comment:
Do we need to request new HMPPS Auth clients for this, and reference the credentials in the values.yaml file?